Seminar3/ThirdSeminar/src/main/java/sopt/org/ThirdSeminar/service/AlarmService.java

```diff
@@ -0,0 +1,53 @@
+package sopt.org.ThirdSeminar.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import sopt.org.ThirdSeminar.controller.dto.response.AlarmResponseDto;
+import sopt.org.ThirdSeminar.domain.Alarm;
+import sopt.org.ThirdSeminar.domain.User;
+import sopt.org.ThirdSeminar.infrastructure.AlarmRepository;
+import sopt.org.ThirdSeminar.infrastructure.UserRepository;
+import sopt.org.ThirdSeminar.utils.mapper.AlarmMapper;
+
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class AlarmService {
+    private final AlarmRepository alarmRepository;
+    private final UserRepository userRepository;
+    @Transactional
+    public Long createAlarm(Long id, String name) {
+        User user = userRepository.findUserById(id);
+
+        Alarm alarm = Alarm.builder()
+                .user(user)
+                .name(name)
+                .build();
+
+        alarmRepository.save(alarm);
+
+        return alarm.getId();
+    }
+
+    public List<AlarmResponseDto> getAlarmByUserId(Long userId) {
+        List<Alarm> alarmList = alarmRepository.findByUserId(userId);
+
+        ArrayList<AlarmResponseDto> alarmResponseDtoList = new ArrayList<>();
+        for(Alarm alarm : alarmList) {
+            alarmResponseDtoList.add(
+                    AlarmMapper.INSTANCE.ToDto(alarm)
+            );
+        }
```

Comment:
```suggestion
    alarmList.stream()
        .map(AlarmMapper.INSTANCE::ToDto)
        .collect(Collectors.toList());
```
이 map 형식이 잘 될지는 저도 확신은 안서지만 stream을 사용한다면 이쁘게 매핑할 수 있을 것 같습니다!